src/panels/lovelace/cards/hui-tile-card.ts

```diff
@@ -411,6 +451,13 @@ export class HuiTileCard extends LitElement implements LovelaceCard {
     }
     hui-card-features {
       --feature-color: var(--tile-color);
+      padding: 0 12px 12px 12px;
+    }
+    .container.horizontal hui-card-features {
+      flex: 1;
+      --feature-height: 36px;
+      padding: 10px;
```

Comment:
```suggestion
      padding: padding: 12px;
```

I would also add this for `.content` because it fits in our 8px raster.

Reply:
Looks like you noticed the padding inconsistencies here. I filed a bug for it:
https://github.com/home-assistant/frontend/issues/24505